lib/Importer/Caffe2.cpp

```diff
@@ -445,6 +449,44 @@ void caffe2ModelLoader::loadWeights(caffe2::NetDef &net) {
       continue;
     }
 
+    if (typeName == "UniformFill") {
+      /*
+       output: "fc/w"
+       name: ""
+       type: "UniformFill"
+       arg {
+         name: "max"
+         f: 0.25
+       }
+       arg {
+         name: "shape"
+         ints: 1
+         ints: 16
+       }
+       arg {
+         name: "min"
+         f: -0.25
+       }
+      */
+      const auto &name = op.output(0);
+      auto *T = new Tensor();
+      tensors_[name] = T;
+      auto dim = getShape(dict["shape"]);
+      T->reset(ElemKind::FloatTy, dim);
+      auto TH = T->getHandle<>();
+      float tensorMin = loadFloat(dict["min"]);
+      float tensorMax = loadFloat(dict["max"]);
+
+      llvm::outs() << "The model contains UniformFill operator, which generates"
```

Comment:
Do we want to always print this? Maybe use `NDEBUG` to only print on Debug builds?

Reply:
Done.